src/Forms/iOS/ArcGISRuntime.Xamarin.Forms.iOS.csproj

```diff
@@ -143,12 +143,15 @@
     <!-- References -->
     <Reference Include="Esri.ArcGISRuntime, Version=100.2.0.0, Culture=neutral, PublicKeyToken=8fc3cc631e44ad86, processorArchitecture=MSIL">
       <HintPath>..\..\..\packages\Esri.ArcGISRuntime.Xamarin.iOS.100.2.0\lib\Xamarin.iOS10\Esri.ArcGISRuntime.dll</HintPath>
+      <Private>True</Private>
```

Comment:
I don't think any of these changes are relevant to the sample. We should remove them.